x/iro/keeper/create_plan.go

```diff
@@ -129,6 +130,11 @@ func (k Keeper) CreatePlan(ctx sdk.Context, allocatedAmount math.Int, start, pre
 		return "", err
 	}
 
+	// 1 reserved token amount
+	// FIXME: make const
+	reserve := math.NewIntWithDecimal(1, int(rollapp.GenesisInfo.NativeDenom.Exponent))
```

Comment:
I think the `reserve` concept is confusing/ambigious as rollapp token charging acts exactly as the dym creation fee - charged and being put into a pool. so I'd stick with creation fee (for both dym and rollapp token). 

Also - wouldn't it be worth just adding a const now? what's the effort here?